### PR TITLE
Add TIGEM config file

### PR DESCRIPTION
**Pull request by giusmar**
---
name: TIGEM config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: docs/tigem.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# nf-core/configs: TIGEM configuration
+
+To use, run the pipeline with `-profile tigem`. This will download and launch the tigem.config which has been pre-configured with a setup suitable for the TIGEM personal biocluster.
+
+---
+
+For this pipeline you need a docker container containing the tools needed for the specific pipeline. You can find the docker containers on giusmar docker hub profile. A docker image containing the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```
Comment (apeltzer) [inline docs/tigem.md:7]: Maybe reformulate this a bit - it reads as if the user has to do anything (which they really don't apart from waiting) ;-)

**Review by apeltzer — APPROVED** [review]
Looks good to me - consider reformulating that one sentence maybe to improve it for beginners 👍🏻

File: conf/tigem.config [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+singularity {
+    enabled = true
+    autoMounts = true
+    cacheDir = 'work/singularity'
+}
+
+timeline {
+  enabled = true
+  file = "${params.outdir}/execution/timeline.html"
```
Comment (apeltzer) [inline conf/tigem.config:18]: This is not defined, so the tests fail - you have to adjust it :-)

**Review by apeltzer — CHANGES_REQUESTED** [review]
The `params.outdir` part is undefined, need to define this first 👍🏻

File: docs/tigem.md (hunk 1 above)
Comment (giusmar) [reply docs/tigem.md:7]: I want to use a different docker container for each different pipeline, but it is for my configuration.Exactly, nf-core pipelines set a different docker container for each process automatically, so, basically, the better reformulation is to delete the sentence :)
I ll update the doc with further details for 
`process.executor = 'slurm'
google.zone = 'europe-west1'`

File: docs/tigem.md [hunk 3]
```diff
@@ -4,4 +4,4 @@ To use, run the pipeline with `-profile tigem`. This will download and launch th
 
 ---
 
-For this pipeline you need a docker container containing the tools needed for the specific pipeline. You can find the docker containers on giusmar docker hub profile. A docker image containing the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+This pipeline can be used on TIGEM clusters, in which is installed slurm as job scheduling system, so you can use this config if you have the same tool installed. An additional parameter is google.zone to allow downloading data from GCP for a specific time zone. It should not interfere with a local or an AWS configuration.
```
Comment (apeltzer) [inline docs/tigem.md:7]:
```suggestion
This configuration profile can be used on TIGEM clusters, with the pre-installed SLURM job scheduling system. An additional parameter is `google.zone` to allow downloading data from GCP for a specific time zone. It should not interfere with any local or other AWS configuration.
```

Comment (giusmar) [reply docs/tigem.md]: Thank you so much

**Comment by giusmar** [comment]
> The `params.outdir` part is undefined, need to define this first 👍🏻

I removed the code for params outdir. is it still necessary for the merging?

File: docs/tigem.md (hunk 3 above)
Comment (apeltzer) [reply docs/tigem.md]: No this should be fine now.

**Review by apeltzer — APPROVED** [review]

**Pull request merged**